hack/common.sh

```diff
@@ -213,6 +211,7 @@ os::build::internal::build_binaries() {
       fi
 
       local platform_gotags_envvar=OS_GOFLAGS_TAGS_$(echo ${platform} | tr '[:lower:]/' '[:upper:]_')
+      local platform_gotags_test_envvar=OS_GOFLAGS_TAGS_TEST_$(echo ${platform} | tr '[:lower:]/' '[:upper:]_')
```

Comment:
If I were passing the platform-specific `OS_GOFLAGS_TAGS_LINUX` today, that would get used in the build of tests. You've changed that so I need to pass `OS_GOFLAGS_TAGS_TEST_LINUX` for that. This is a breaking change. I'm not sure we have a good way to identify if someone downstream will be broken by such a change.